tvb/simulator/plot/config.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
```

Comment:
This configr I would not like to see copied entirely here.
Could we deal only with FiguresConfig in a slightly modified form, thus we also know ehere to store figures (so part of the OutputConfig merge in the same class)

Reply:
I think, yes. A possible solution can be seen in a commit.